src/components/about-us/flippingCard.jsx

```diff
@@ -0,0 +1,28 @@
+import { useTranslations } from 'next-intl';
+import React from 'react';
+import styles from "@/styles/components/aboutus/flippingCard.module.scss";
+export async function getStaticProps({ locale }) {
+    return {
+        props: {
+            messages: (await import(`@/messages/${locale}.json`)).default
+        }
+    };
+}
+
+export default function PopupCard({ content, image, title, alt }) {
+    const t = useTranslations('Flipping Cards');
+    return (
+        <div className="container">
+            <div className='FlipCard'>
```

Comment:
You can essentially get rid of the `container` div here since it's added as an extra wrapper around everything else that isn't really doing much.